docs/source/user/generating_users.rst

```diff
@@ -25,10 +25,14 @@ This command generates users whose usernames start with a particular prefix. Usa
 
 Note that the expiration and password-length are optional parameters which default to never and 8 respectively.
 
+In the response, you can find the field ``user_credentials`` containing the list of users created
+(example: [['username', 'password'], ['sample_user', 'BBuOb5sN']]") and field ``pdf_link``
+containing a download link to a PDF file containing user credentials.
```

Comment:
what if we change the command so it returns the pdf as its output and we change the instruction to suggest doing

```
./manage.py command > output.pdf
```

I think this is more unix friendly and more practical for most users.

Reply:
I made a mistake here, It's suppose, I have made some changes, it used to say `Generated a batch of users with prefix <given prefix>`; I have reverted to this output, do you think it's better to output the pdf instead?

Reply:
we have to think like users, if you were using this feature to generate users and get the credentials, what would you prefer?
I think I would prefer to have the PDF.